float32[] transform_coordinate
---
float32[] tranformed_coordinate
